sources/reference/admiral-aws-marketplace.md

```diff
@@ -7,100 +7,120 @@ page_keywords: install, microservices, Continuous Integration, Continuous Deploy
+  for the `Messaging` and `State` components in the `Initialize Infrastructure` 
+  section. These must be 8 characters or more.
 
-- The `Installer Access Key` and `Installer Secret Key` fields can be left
-  blank.
+- The `Installer Access Key` and `Installer Secret Key` fields are not required
+  for the Startup Edition and can be left blank.
 
 - Once passwords are set, click `Initialize`. This will prep the system with
   core dependencies like database, message queue, secrets store, docker etc.
 
+### Configure & Install
 - After initialization is complete, navigate to the next section `Configure and
-  Install` to set up authentication provider(s).
-
-- In the `Authorization and Source Control Management (SCM) Providers`,
-  section, select the SCM provider of choice. We'll configure Github as
-  a provider as an example.
-  - Check the box for `Github`, `Auth` column
-  - On `https://github.com/settings/developers` register a new application
+  Install` to set up your authentication provider(s). Shippable leverages one 
+  of your SCM providers to manage auth/identity.
+
+- In the `Authorization and Source Control Management (SCM) Providers` section,
+  section, select the SCM provider of choice. We'll configure GitHub as a
+  provider as an example.
+  - Check the box for `GitHub`, `Auth` column
+  - Go to `https://github.com/settings/developers` and register a new
+    application
   - The callback URL should be `http://1.2.3.4:50001/auth/github`
-  - Save and copy the keys for this application into Github auth section of
-    Shippable admin panel
+  - Save and copy the keys for this application into GitHub auth section of
+    Shippable Admin panel
 
-- In the `Service Addresses` change the `API` IP address to the public IP
-  of server. The port should remain the same. For our example, the value will
-  be `http://1.2.3.4:50000`
+- In the `Service Addresses` section, change the `API` IP address to the public
+  IP of server. The port should remain the same. For our example, the value
+  will be `http://1.2.3.4:50000`
 
-- In the `Service Addresses` change the `WWW` IP address to the public IP
-  of server. The port should remain the same. For our example, the value will
-  be `http://1.2.3.4:50001`
+- In the `Service Addresses` section, change the `WWW` IP address to the public
+  IP of server. The port should remain the same. For our example, the value
+  will be `http://1.2.3.4:50001`
 
 - Click `Install`. This should set up Shippable services and bring up the UI.
-  Once complete, you should be able to navigate to `http://1.2.3.4:50001` and
-  log into Shippable using a Github account.
-
+  Once complete, navigate to `http://1.2.3.4:50001` and log into Shippable 
+  using a GitHub account.
 
-### Setup superuser account
+### Add-ons
 
-Shippable UI also provides an admin panel which helps users to
+In addition to the Shippable Server Admin panel, within the product UI, 
```

Comment:
This sounds like "Shippable Server Admin panel" is in the product UI.  How about `In addition to the Shippable Server Admin panel, Shippable also provides an Admin panel within the product UI to:`?